43810.7 <c-14#256> Created logger
43810.7 <c-14#256> appl_initialize was called
43810.7 <c-14#256:9> Logger was updated
43810.7 <c-14#256:9> Started intializing cube lib
43810.7 <c-14#256:9> Finished intializing cube lib with worker, starting to work
43810.7 <c-14#256:9> requestCubes => [9] Buffer size: 0
43810.7 <c-14#256:9> Sent requestCubes signal to root
43810.7 <c-14#256:9> Received 1 cubes from root
43810.7 <c-14#256:9> The main loop continues
43810.7 <c-14#256:9> Started solving a cube
43895.2 <c-14#256:9> Cube failed
43895.2 <c-14#256:9> Added failed cube
43895.2 <c-14#256:9> returnFailedCubes => [9] Buffer size: 5
43895.2 <c-14#256:9> Sent 1 failed cubes to root
43895.2 <c-14#256:9> Received receivedFailedCubes signal from root
43895.2 <c-14#256:9> requestCubes => [9] Buffer size: 0
43895.2 <c-14#256:9> Sent requestCubes signal to root
43895.2 <c-14#256:9> Received 1 cubes from root
43895.2 <c-14#256:9> The main loop continues
43895.2 <c-14#256:9> Started solving a cube
43936 <c-14#256:9> appl_interrupt was called
43936 <c-14#256:9> Interrupting worker
43936 <c-14#256:9> STOP (0.00s since last cb)
43936 <c-14#256:9> Started withdraw thread
43936 <c-14#256:9> appl_interrupt was called
43936 <c-14#256:9> appl_withdraw was called
43936 <c-14#256:9> Solving interrupted
43936 <c-14#256:9> The main loop continues
43936 <c-14#256:9> Leaving the main loop due to an interruption
43936 <c-14#256:9> Finished withdrawing lib
43936 <c-14#256:9> Enter destructor of CubeWorker
43936 <c-14#256:9> Time waiting for messages: 0.000
43936 <c-14#256:9> Enter destructor of CubeWorkerInterface
43936 <c-14#256:9> Enter destructor of CubeRoot
43936 <c-14#256:9> Finished withdraw thread
43997.4 <c-14#256:9> Enter destructor
43997.4 <c-14#256:9> Time working: 125.312
43997.4 <c-14#256:9> Time suspended: 0.000
43997.4 <c-14#256:9> Joined cleanup thread
43997.4 <c-14#256:9> Exit destructor
43997.4 <c-14#256:9> Destructing logger
